docker/Dockerfile

```diff
@@ -25,10 +26,9 @@ COPY ./docker/node/*.js ./
 
 EXPOSE 8080
 
-RUN poetry install .
-RUN poetry shell .
+RUN pip install .
```

Comment:
Using `poetry install` ensures that dependencies are version locked, which `pip install .` doesn't. 

You can run `poetry config virtualenvs.create false` before any other poetry commands to prevent it from creating/managing a virtualenv, so you don't need to `poetry run ...`.

Reply:
Ok poetry run was a failure.
I just wanted it to build, went back to pip.

Reply:
Please notice that

`poetry install .`

give a "too many argument" error.

Reply:
And 

`poetry shell .`

cannot be executed.

Reply:
Yeah I think Arvid killgissade there :)

It should just be `poetry install` and then `python whatever.py` once you've added the `poetry config virtualenvs.create false` line.